src/lib/Libifl/pbs_loadconf.c

```diff
@@ -919,6 +924,17 @@ __pbs_loadconf(int reload)
 			goto err;
 		}
 	}
+	if (pbs_conf.cp_path == NULL) {
```

Comment:
Before this shouldn't we check for "PBS_CP" environment variable to get its value as we do for other pbs.conf key-value?

Reply:
Yes. Good catch.